From 5b3e5314e618f0d1e381709ee45a35f0346390be Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Wed, 19 Aug 2015 15:24:24 +0800
Subject: [PATCH 0242/5242] MLK-11290-02 ARM: dts: imx7d-12x12-lpddr3-arm2:
 add enet dts file to makefile

commit  bb471230edc45af615cf182be6eda9c3d9970b60 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add enet dts file to makefile.

Signed-off-by: Fugang Duan <B38611@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/Makefile |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index c5b87b0..4e862c4 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -561,7 +561,8 @@ dtb-$(CONFIG_SOC_IMX7D) += \
 	imx7s-warp.dtb \
 	imx7d-12x12-lpddr3-arm2.dtb \
 	imx7d-12x12-lpddr3-arm2-m4.dtb \
-	imx7d-12x12-lpddr3-arm2-ecspi.dtb
+	imx7d-12x12-lpddr3-arm2-ecspi.dtb \
+	imx7d-12x12-lpddr3-arm2-enet2.dtb
 dtb-$(CONFIG_SOC_LS1021A) += \
 	ls1021a-moxa-uc-8410a.dtb \
 	ls1021a-qds.dtb \
-- 
1.7.9.5

